third_party/lua/luadebug.lua

```diff
@@ -687,8 +709,7 @@ end
 
 -- Detect Lua/LuaJIT version.
 LUA_JIT_SETLOCAL_WORKAROUND = -1
-dbg_writeln(COLOR_YELLOW .. "luadebug.lua: " .. COLOR_RESET .. "Loaded for " ..
-            jit.version)
+dbg_writeln(color_yellow(DEBUGGER .. ": ") .. "Loaded for " .. jit.version)
 jit.off()
```

Comment:
If you want to disable JIT at all here, you also need to call `jit.flush` to purge the existing traces.

Reply:
But wait a minute, LuaJIT documentation says:

> jit.off disables JIT compilation for a Lua function and flushes any already compiled code from the code cache. 

Which flushing here is actually meant?